.travis.yml

```diff
@@ -43,12 +43,12 @@ before_install:
 
 install:
    - pip3 install wheel
-   - docker pull jamesmcclain/jupyter-geopyspark:stage0
+   - docker pull jamesmcclain/jupyter-geopyspark:aws-build-gdal
```

Comment:
Not published from scripts but I suppose its safe to publish this once and use as resource after image rename.